core/arch/x86/decode_table.c

```diff
@@ -5567,11 +5567,11 @@ const instr_info_t e_vex_extensions[][3] = {
     {INVALID, 0x663a0518, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
   }, { /* e_vex ext 73 */
     {INVALID,   0x663a0618, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
-    {OP_vperm2f128, 0x663a0618, "vperm2f128", Vx, xx, Hx,Wx, Ib, mrm|vex|reqp, x, END_LIST},
+    {OP_vperm2f128, 0x663a0618, "vperm2f128", Vqq, xx, Hqq, Wqq, Ib, mrm|vex|reqp, x, END_LIST},
     {INVALID, 0x663a0618, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
   }, { /* e_vex ext 74 */
     {INVALID,   0x663a1818, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
-    {OP_vinsertf128, 0x663a1818, "vinsertf128", Vx, xx, Hx,Wx, Ib, mrm|vex|reqp, x, END_LIST},
+    {OP_vinsertf128, 0x663a1818, "vinsertf128", Vqq, xx, Hqq, Wqq, Ib, mrm|vex|reqp, x, END_LIST},
```

Comment:
The 3rd arg is xmm-sized so Wdq, not Wqq.

Please add a test with a register arg there so we'll catch this (should then not match other decoders).

Reply:
You're right. Even the Intel tables are wrong (Table A-5).

Reply:
Please look at the Intel details page.